examples/customid/domain/id.go

```diff
@@ -0,0 +1,24 @@
+package domain
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+)
+
+// ID64 implements ID interface
+type ID64 int64

```

Comment:
Would be nice if you would move UUID there too, but I can also do that later if you want.